interview-questions.md

```diff
@@ -7,63 +7,80 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is a group of statements that preform a certain task.
 
-  Researched answer:
+  Researched answer: Functions are reusable pieces of code that are usually called upon a lot. They also
+  can have information pushed through what we call an argument.
 
 
 
 2. What is the difference between .map() and .filter()?
 
-  Your answer:
+  Your answer: A .map goes through an array and returns one with the same length. A .filter
+  filters out some type of value.
 
-  Researched answer:
+  Researched answer: So .map goes through an array and returns another one with the
+  the same length, .filter does almost the same thing but only returns the
+  ones with true statement that you decided.
 
 
 
 3. What is the difference between console.log() and return?
 
-  Your answer:
+  Your answer: console.log(), is like your "go" button because without it a lot of arrays won't be able to run.
```

Comment:
Not quite, regarding the 'go' button. Console.log doesn't really do anything to arrays or anything like that. Its simply a debugging tool that prints whatever you pass into it to the terminal and ONLY the terminal.  Code still runs without console.log being used, its just not printed to the terminal.